skymp5-server/cpp/papyrus_vm_lib/Reader.cpp

```diff
@@ -118,6 +121,7 @@ StringTable Reader::FillStringTable()
   std::vector<std::string> storage;
 
   int SizeStringTable = Read16_bit();
+  storage.reserve(SizeStringTable);
```

Comment:
This one also should be fixed even if this variable wasn't created by this PR. I think the more convenient name would be `stringTableSize`